core-feature-pack/src/main/resources/modules/system/layers/base/org/jboss/as/domain-add-user/main/module.xml

```diff
@@ -34,6 +34,7 @@
     </resources>
 
     <dependencies>
+        <module name="org.aesh"/>
```

Comment:
Was the org.aesh import really needed in the end on this one?  This module is predominantly a dependency on org.jboss.as.domain-management with a main class specified.

Reply:
Yes, You are right. I am sorry I didn't noticed this. I fixed it along with the other suggestions from your comments. Thank You.